dev-tools/mage/pkgtypes.go

```diff
@@ -500,12 +501,15 @@ func copyInstallScript(spec PackageSpec, script string, local *string) error {
 }
 
 func (s PackageSpec) hash() string {
-	h, err := hashstructure.Hash(s, nil)
+	out, err := yaml.Marshal(s)
```

Comment:
How big are these on average? Relative to `hashstructure`, we'd be using more memory here, as we need to hold the full yaml encoding of the spec in memory.

Reply:
This is only used for packaging the elastic-agent and is not something even used by a running elastic-agent.  Even with the increase in memory here it would be very minimal, most package specifications are not that large.